media/css/firefox/new/trailhead/download.scss

```diff
@@ -43,10 +43,11 @@ $image-path: '/media/protocol/img';
             border: none;
             color: #fff;
             cursor: pointer;
-            display: inline-block;
+            text-decoration: underline;
+            display: ruby;
```

Comment:
Curious why is `display: ruby` needed here?

Reply:
Hi @alexgibson, In the begging I thought this issues was related with display propertie, so I was testing different options, and trying to understand the behavior. I left there because this option solve the problem in Firefox, but not in Chrome. Then, I was use white-space: nowrap, it worked on Chrome, and all the other browsers. 

But like you pointed out, this white-space: nowrap options cuts  on smaller screens. Sorry about that!